src/UnityExtension/Assets/Editor/GitHub.Unity/UI/BranchesView.cs

```diff
@@ -272,6 +272,7 @@ private void OnButtonBarGUI()
                             {
                                 if (success)
                                 {
+                                    EntryPoint.ApplicationManager.UsageTracker.IncrementNumberOfLocalBranchCreations();
```

Comment:
The base class has a `IApplicationManager Manager` property, we should use that and avoid static calls to `EntryPoint` wherever possible.